lib/chef/provider/aws_vpc.rb

```diff
@@ -177,8 +177,9 @@ def destroy_aws_object(vpc)
     if ig
       Cheffish.inline_resource(self, action) do
         aws_internet_gateway ig do
-          ig_tag = ig.tags.find{|i|i.key=='OwnedByVPC'}.value
-          if ig_tag == vpc.id
+          ig_tag = ig.tags.find{|i|i.key=='OwnedByVPC'}
```

Comment:
This line could be replaced by ig_tag = ig.tags.find { |i|i.key == "OwnedByVPC" }